aspnetcore/blazor/security/includes/httpcontext.md

```diff
@@ -1,5 +1,38 @@
+:::moniker range=">= aspnetcore-8.0"
+
+<xref:Microsoft.AspNetCore.Http.IHttpContextAccessor> must be avoided when interactive rendering for the following reasons:
+
+* The interface isn't implemented for interactive WebAssembly rendering.
+* For interactive server rendering, the implementation isn't guaranteed to exist.
```

Comment:
```suggestion
<xref:Microsoft.AspNetCore.Http.IHttpContextAccessor> must be avoided with interactive rendering because there isn't a valid `HttpContext` available.
```